sys-boot/plymouth/plymouth-9999.ebuild

```diff
@@ -1,113 +1,118 @@
 SRC_URI="https://dev.gentoo.org/~aidecoe/distfiles/${CATEGORY}/${PN}/gentoo-logo.png"
 
 if [[ ${PV} == 9999 ]]; then
 	inherit git-r3
 	EGIT_REPO_URI="https://gitlab.freedesktop.org/plymouth/plymouth"
 else
 	SRC_URI="${SRC_URI} https://www.freedesktop.org/software/plymouth/releases/${P}.tar.xz"
-	KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~ia64 ~ppc ~ppc64 ~riscv ~sparc ~x86"
+	KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~ia64 ~loong ~ppc ~ppc64 ~riscv ~sparc ~x86"
 fi
 
-inherit autotools readme.gentoo-r1 systemd
-
 DESCRIPTION="Graphical boot animation (splash) and logger"
 HOMEPAGE="https://cgit.freedesktop.org/plymouth/"
 
-LICENSE="GPL-2"
+LICENSE="GPL-2+"
 SLOT="0"
-IUSE="debug +drm +gtk +pango selinux +split-usr static-libs +udev"
+IUSE="debug +drm +gtk +pango selinux freetype +split-usr +udev doc upstart-monitoring systemd"
+
+BDEPEND="
+	app-text/docbook-xsl-stylesheets
+	>=dev-build/meson-0.62
+	dev-libs/libxslt
+	virtual/pkgconfig
+"
 
-CDEPEND="
+DEPEND="
+	dev-libs/libevdev
 	>=media-libs/libpng-1.2.16:=
+	virtual/libc
+	x11-libs/libxkbcommon
+	x11-misc/xkeyboard-config
 	drm? ( x11-libs/libdrm )
+	elibc_musl? ( sys-libs/rpmatch-standalone )
+	freetype? ( media-libs/freetype:2 )
 	gtk? (
 		dev-libs/glib:2
-		>=x11-libs/gtk+-3.14:3
 		x11-libs/cairo
+		>=x11-libs/gtk+-3.14:3[X]
+	)
+	pango? (
+		x11-libs/cairo
+		>=x11-libs/pango-1.21
+	)
+	systemd? ( sys-apps/systemd )
+	udev? ( virtual/libudev )
+	upstart-monitoring? (
+		sys-apps/dbus
+		sys-libs/ncurses
 	)
-	pango? ( >=x11-libs/pango-1.21 )
-"
-DEPEND="${CDEPEND}
-	elibc_musl? ( sys-libs/rpmatch-standalone )
-	app-text/docbook-xsl-stylesheets
-	dev-libs/libxslt
-	virtual/pkgconfig
 "
-# Block due bug #383067
-RDEPEND="${CDEPEND}
+
+RDEPEND="${DEPEND}
 	selinux? ( sec-policy/selinux-plymouthd )
 	udev? ( virtual/udev )
-	!<sys-kernel/dracut-0.37-r3
 "
 
 DOC_CONTENTS="
 	Follow the following instructions to set up Plymouth:\n
 	https://wiki.gentoo.org/wiki/Plymouth#Configuration
 "
 
-PATCHES=(
-	"${FILESDIR}"/0.9.3-glibc-sysmacros.patch
-)
-
 src_prepare() {
 	use elibc_musl && append-ldflags -lrpmatch
 	default
-	eautoreconf
 }
 
 src_configure() {
-	local myconf=(
-		--with-system-root-install=no
-		--localstatedir=/var
-		--without-rhgb-compat-link
-		--enable-documentation
-		--enable-systemd-integration
-		--with-systemdunitdir="$(systemd_get_systemunitdir)"
-		$(use_enable !static-libs shared)
-		$(use_enable static-libs static)
-		$(use_enable debug tracing)
-		$(use_enable drm)
-		$(use_enable gtk)
-		$(use_enable pango)
-		$(use_with udev)
+	local emesonargs=(
+		--localstatedir "${EPREFIX}"/var
+		$(meson_feature gtk)
+		$(meson_feature freetype)
+		$(meson_feature pango)
+		$(meson_feature udev)
+		$(meson_use drm)
+		$(meson_use systemd systemd-integration)
+		$(meson_use doc docs)
+		$(meson_use debug tracing)
+		$(meson_use upstart-monitoring)
 	)
-	econf "${myconf[@]}"
+	meson_src_configure
 }
 
-src_install() {
-	default
+src_compile() {
+	meson_src_compile
+}
```

Comment:
This is the default `src_compile` when `meson.eclass` is inherited. So no need to specify this manually.